### ISSUE TITLE
Copy entry between 2 open databases not working

### ISSUE THREAD

**Issue by AlbertBaubleDeem** [keepassxreboot/keepassxc#8974]
## Overview
Copying an entry between 2 open databases does not work.

I tried copying an entry using this advice: [droidmonkey](https://github.com/droidmonkey) commented [on Apr 12, 2021](https://github.com/keepassxreboot/keepassxc/issues/6397#issuecomment-817395160)

> To move, just drag an entry to the other database and drop it on the desired group. To copy, drag then press and hold CTRL, drop on desired group.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1.  Open database1
2. Open database2
3. Select an entry from database1, holding left mouse button, and drag it to the open tab of database2 and into a folder in it. Press CTRL and drop it to the desired folder.

As the behaviour manifested itself on macOS, I tried the above steps with both CTRL and COMMAND buttons but did not notice any difference in behaviour.

## Expected Behavior
The database1 entry is copied to the database2.

## Actual Behavior
The database1 entry is moved to the database2.

## Context

KeePassXC - Version 2.7.1
Revision: 5916a8f

Qt 5.15.2
Debugging mode is disabled.

Operating system: macOS 12.6
CPU architecture: x86_64
Kernel: darwin 21.6.0

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare
- YubiKey
- Quick Unlock

Cryptographic libraries:
- Botan 2.19.1

Operating System: macOS Monterey 12.6.2 (21G320)

**Comment by droidmonkey** [comment]
Did you try using Alt/option?

**Comment by michaelk83** [comment]
Not sure, but maybe you need to press and hold Ctrl/etc *before* you start dragging?

**Comment by droidmonkey** [comment]
I agree I couldn't make it work either with any key combo in any order.

**Comment by ralfw2** [comment]
I can confirm that this does not (or no longer) work for Windows 10 too. Tried Keypass 2.52 (64 bit first), then upgraded to 2.53, same issue. IMHO it would also be helpful to have a "copy entry" function in the right mouse click context menu, in the group menu then a "paste entry" fuction would be helpful (regardless in which database TAB you are). Thanks for considering.

**Comment by michaelk83** [comment]
@ralfw2 KeePass and KeePassXC are two different programs (they're compatible, but separate). This forum is for KeePassXC issues. KeePass is at https://keepass.info/

**Comment by dmaslenko** [comment]
According to the [MacOs User Guide](https://support.apple.com/guide/mac-help/drag-and-drop-items-mh35852/13.0/mac/13.0):

> To copy the item instead of moving it, press and hold the **Option** key while you drag.

But even with **Option** it does not work.
I tried on the Linux Ubuntu - it does not work with any combination as well.
Did not check on Windows.

@droidmonkey if you did not start this task you may reassign it to me, I will play...
Also, it makes sense to tag the issue by `user interface` without platform specific.

**Comment by droidmonkey** [comment]
Yes please take a crack at it

**Comment by dmaslenko** [comment]
After more detailed investigation I realized I tried to drop into the entries table rather tree folder as was mentioned in the ticket description.
The dropping into folder works as expected on both Linux (with `Ctrl`) and MacOS (with `Option`).
The below recorded video on MacOS (Monterey 12.6.3) shows it.
Note, before dropping I pressed the `Option` key and the corresponding **green plus icon** displayed:
![Drag-drop-copy-macos](https://user-images.githubusercontent.com/121412908/216531754-a4bc5721-d009-4ced-b20e-fbc93f514c68.gif)

The same behaviour we have on the Linux but with `Ctrl` key.
So, I propose to close the ticket as not a bug.
